### PR TITLE
🌐 Add Chinese translation for Tutorial - Schema Extra Example

### PR DESCRIPTION
**Pull request by maoyibo**
Chinese translation add tutorial/schema-extra-example

### CODE REVIEW

File: docs/zh/docs/tutorial/schema-extra-example.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+```Python hl_lines="15-23"
+{!../../../docs_src/schema_extra_example/tutorial001.py!}
+```
+
+这些额外的信息将按原样添加到输出的JSON模式中。
+
+## `Field` 的附加参数
+
+在 `Field`, `Path`, `Query`, `Body` 和其他你之后将会看到的工厂函数，你可以为JSON 模式声明额外信息，你也可以通过给工厂函数传递其他的任意参数来给JSON 模式生命额外信息，比如增加 `example`:
```
Comment (waynerv) [inline docs/zh/docs/tutorial/schema-extra-example.md:21]: 生命 -> 声明

File: docs/zh/docs/tutorial/schema-extra-example.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+```Python hl_lines="4  10-13"
+{!../../../docs_src/schema_extra_example/tutorial002.py!}
+```
+
+!!! warning
+    请记住，传递的那些额外参数不会添加任何验证，只会添加注释，用于文档目。
```
Comment (waynerv) [inline docs/zh/docs/tutorial/schema-extra-example.md:28]: 用于文档目。->用于文档的目的

File: docs/zh/docs/tutorial/schema-extra-example.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+使用上面的任何方法，它在 `/docs` 中看起来都是这样的:
+
+<img src="/img/tutorial/body-fields/image01.png">
+
+## 技术细节
+
+关于 `example` 和 `examples`...
+
+JSON Schema在最近的一个版本中定义了一个字段 <a href="https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.9.5" class="external-link" target="_blank">`examples`</a> ，但是 OpenAPI 基于之前的一个旧版JSON Schema，并没有 `examples`.
```
Comment (waynerv) [inline docs/zh/docs/tutorial/schema-extra-example.md:50]: “most recent versions” should not translated to "最近的一个版本中"

File: docs/zh/docs/tutorial/schema-extra-example.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
+# 额外的模式 - 例子
+
+您可以在JSON模式中定义额外的信息。
+
+一个常见的用例是添加一个将在文档中显示的 `示例`。
```
Comment (waynerv) [inline docs/zh/docs/tutorial/schema-extra-example.md:5]: unnecessary whitespace before `实例`.

**Review by waynerv — CHANGES_REQUESTED** [review]
Note the whitespace between English and Chinese characters.

File: docs/zh/docs/tutorial/schema-extra-example.md (hunk 4 above)
Comment (maoyibo) [reply docs/zh/docs/tutorial/schema-extra-example.md]: Fixed, thanks.

File: docs/zh/docs/tutorial/schema-extra-example.md (hunk 3 above)
Comment (maoyibo) [reply docs/zh/docs/tutorial/schema-extra-example.md]: Fixed, thanks.

File: docs/zh/docs/tutorial/schema-extra-example.md (hunk 2 above)
Comment (maoyibo) [reply docs/zh/docs/tutorial/schema-extra-example.md]: Fixed, thanks.

File: docs/zh/docs/tutorial/schema-extra-example.md (hunk 1 above)
Comment (maoyibo) [reply docs/zh/docs/tutorial/schema-extra-example.md]: Fixed, thanks.

**Comment by waynerv** [comment]
What do you think about the idea of keeping one space between Chinese and English characters(both left and right), since most documents that have been merged so far respect this convention.

File: docs/zh/docs/tutorial/schema-extra-example.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+您可以在JSON模式中定义额外的信息。
+
+一个常见的用例是添加一个将在文档中显示的`示例`。
+
+有几种方法可以声明额外的JSON模式信息。
+
+## Pydantic `schema_extra`
+
+您可以使用 `Config` 和 `schema_extra` 为Pydantic模型声明一个示例，如<a href="https://pydantic-docs.helpmanual.io/usage/schema/#schema-customization" class="external-link" target="_blank">Pydantic's docs: Schema customization</a>中所述:
```
Comment (waynerv) [inline docs/zh/docs/tutorial/schema-extra-example.md:11]: Pydantic's docs: Schema customization -> Pydantic 文档：Schema customization

File: docs/zh/docs/tutorial/schema-extra-example.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
+# 额外的模式 - 例子
+
+您可以在JSON模式中定义额外的信息。
+
+一个常见的用例是添加一个将在文档中显示的`示例`。
```
Comment (waynerv) [inline docs/zh/docs/tutorial/schema-extra-example.md:5]: It seems better not to translate text in code blocks, e.g `example`.

File: docs/zh/docs/tutorial/schema-extra-example.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
+# 额外的模式 - 例子
```
Comment (waynerv) [inline docs/zh/docs/tutorial/schema-extra-example.md:1]: This title is not accurate, for the content below is about extra information in shcema, not extra shcemas.

**Comment by maoyibo** [comment]
> What do you think about the idea of keeping one space between Chinese and English characters(both left and right), since most documents that have been merged so far respect this convention.

I may not pay special attention to this situation in my previous translation, I will pay attention to it in the future, but I think the most important thing is the accuracy of the translation, the whitespace should not affect the generation and reading of the document. We can work on this in future updates.

File: docs/zh/docs/tutorial/schema-extra-example.md (hunk 7 above)
Comment (maoyibo) [reply docs/zh/docs/tutorial/schema-extra-example.md]: I revised the translation to “模式的额外信息”, What do you think of this?

File: docs/zh/docs/tutorial/schema-extra-example.md (hunk 5 above)
Comment (maoyibo) [reply docs/zh/docs/tutorial/schema-extra-example.md]: Fixed, thanks.

File: docs/zh/docs/tutorial/schema-extra-example.md (hunk 6 above)
Comment (maoyibo) [reply docs/zh/docs/tutorial/schema-extra-example.md]: Fixed, thanks.

**Review by waynerv — APPROVED** [review]
LGTM. 👍

**Comment by waynerv** [comment]
> > What do you think about the idea of keeping one space between Chinese and English characters(both left and right), since most documents that have been merged so far respect this convention.
> 
> I may not pay special attention to this situation in my previous translation, I will pay attention to it in the future, but I think the most important thing is the accuracy of the translation, the whitespace should not affect the generation and reading of the document. We can work on this in future updates.

It's OK.

**Review by Laineyzhang55 — APPROVED** [review]

**Comment by tiangolo** [comment]
Awesome! Thanks a lot for your contribution @maoyibo ! :bow: :coffee: 

And thanks @waynerv and @Laineyzhang55 as always for the help and reviews :cake:
